addons/binding/org.openhab.binding.airquality/pom.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.binding</groupId>
+    <artifactId>pom</artifactId>
+    <version>2.0.0-SNAPSHOT</version>
```

Comment:
Oh, by the way, we are at 2.1.0-SNAPSHOT as of four hours ago:

https://github.com/openhab/openhab2-addons/commit/af6fe5b1be64a009693157e733e2cd36952e00ae

I guess I need to update my pull requests as well 😄